cg/cli/workflow/mutant/base.py

```diff
@@ -99,3 +99,39 @@ def start_available(context: click.Context, dry_run: bool = False):
             exit_code = EXIT_FAIL
     if exit_code:
         raise click.Abort
+
+
+@mutant.command("store-available")
+@OPTION_DRY
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
+    """Run QC checks and store bundles for all finished analyses in Housekeeper."""
+
+    analysis_api: MutantAnalysisAPI = context.obj.meta_apis["analysis_api"]
+
+    exit_code: int = EXIT_SUCCESS
+
+    analysis_api.run_qc_and_fail_analyses(dry_run=dry_run)
+
+    for case_obj in analysis_api.get_cases_to_store():
+        LOG.info("Storing deliverables for %s", case_obj.internal_id)
```

Comment:
Minor thing, but we try to use f-strings as much as possible because they are a little easier to read
```python
LOG.info(f"Storing deliverables for {case_obj.internal_id}")
```

Reply:
Added!